px-data-grid-filter-entity.html

```diff
@@ -72,23 +72,6 @@
           selected="{{entity.selectedDateRangeName}}">
         </px-dropdown>
       </div>
-
```

Comment:
Why removed

Reply:
Sorry now I understand, if this was the correct px-date-picker. Can you still maybe <!-- --> it, so it would be easy to add it back later if required?

Reply:
> Can you still maybe keep it it, so it would be easy to add it back later if required?

Yeah, I was thinking about that too, reverted and commented.